tests/schema/tests.py

```diff
@@ -118,6 +119,17 @@ def get_constraints(self, table):
 
     # Tests
 
+    def test_index_name_hash_is_same(self):
+        """
+        Test for https://code.djangoproject.com/ticket/24390
```

Comment:
Doesn't seem to me that a doc string is really needed since the test is simple, but if you think it's useful you could say something useful like "Index names should be deterministic." -- that's more helpful than requiring someone to reference a ticket.